frontend/traditional-games-explorer/style.css

```
@@ -0,0 +1,875 @@
+    cursor: pointer;
+    transition: var(--transition-smooth);
+    animation: gamCardFadeIn 0.45s ease both;
+}
+
+.gam-card:hover {
+    transform: translateY(-6px);
+    border-color: var(--primary-gold);
+    box-shadow: var(--shadow-glow-active);
+}
+
+@keyframes gamCardFadeIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "gamCardFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)